.github/copilot-instructions.md

```
@@ -0,0 +1,49 @@
+- Follow **single-responsibility principle (SRP)** for functions and modules.
+- Use **descriptive variable and function names**.
+- Write **reusable and composable** utility functions.
+- Ensure **all functions have proper return types**.
+- Use **async/await** for handling asynchronous operations.
+- Maintain **consistent import structure**
+- Keep the **codebase modular** by separating logic into appropriate directories.
+- Use libraries like graphology, turf and which ever is necessary for the project.
+
+## Directory Structure
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified
null

(MD040, fenced-code-language)